test/source-updater.test.js

```diff
@@ -356,10 +356,10 @@ QUnit.test('removeAudio does not remove video buffer', function(assert) {
     video: 'avc1.4D001E'
   });
 
-  this.sourceUpdater.appendBuffer({type: 'audio', bytes: mp4Audio()}, () => {
```

Comment:
should this test still not append audio first?

Reply:
🤔

Reply:
Since audio won't be appended until video is, the callback won't be run, I think the test is still testing the same thing though since it always waited for both callbacks anyway.

Reply:
Ah, I see. Is it possible to make a test that exercises delaying appending the audio buffer?

Reply:
Yeah let me right a test for the new behavior

Reply:
Done